google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/integration/AppEngineServerHelper.java

```diff
@@ -32,14 +33,14 @@
   public ApplicationServerInfo getApplicationServerInfo(
       ApplicationServerPersistentData persistentData)
       throws CantFindApplicationServerJarsException {
-    final AppEngineSdk sdk = ((AppEngineServerData) persistentData).getSdk();
-    String version = sdk.getVersion();
-    return new ApplicationServerInfo(sdk.getLibraries(),
+    final CloudSdkService sdkService = CloudSdkService.getInstance();
+    String version = sdkService.getVersion();
+    return new ApplicationServerInfo(sdkService.getLibraries(),
         "AppEngine Dev" + (version != null ? " " + version : ""));
   }
 
   public ApplicationServerPersistentData createPersistentDataEmptyInstance() {
-    return new AppEngineServerData("");
```

Comment:
Good point. The config subclass (AppEngineServerData) was removed. This whole class can actually be deleted too as its not doing anything.